Simplified/LibraryFinder/NYPLLibraryFinderBusinessLogic.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  NYPLLibraryFinderBusinessLogic.swift
+//  Simplified
+//
+//  Created by Ernest Fan on 2021-04-21.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+protocol NYPLLibraryFinderDataProviding {
+  var userAccounts: [Account] { get }
+  var newLibraryAccounts: [Account] { get }
+  
```

Comment:
Are userAccounts and newLibraryAccounts going at change with the state of the class? If not, can we have them as constants instead?